OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -282,7 +282,7 @@ private boolean parseEnrollLinkAndCallActionListener(@Nullable String strUrl) {
         /**
          * Callback that indicates error during page load.
          */
-        void onPageLoadError();
+        void onPageLoadError(WebView view, int errorCode, String description, String failingUrl);
```

Comment:
Now that we have updated `onPageLoadError` to match the deprecated function `onReceivedError(WebView view, int errorCode, String description, String failingUrl)`'s prototype, we should also override the `onReceivedHttpError(WebView view, WebResourceRequest request, WebResourceResponse errorResponse)` in this class and create a matching `onPageLoadError` function in `IPageStatusListener` interface.